src/LcovGeneratorTests/LcovGeneratorTests.csproj

```diff
@@ -0,0 +1,20 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
```

Comment:
Add
```xml
 <PropertyGroup>
    <VstestCoverageRoot Condition="$(VstestCoverageRoot) == ''">..\..\</VstestCoverageRoot>
    <TestProject>true</TestProject>
  </PropertyGroup>
```
Remove
```xml
    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
    <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
```

Reply:
also needed to add  <Import Project="$(VstestCoverageRoot)scripts/build/Vstest.Coverage.Settings.targets" />